op-monitorism/global_events/rules/rules_mainnet_L1/Initialized-Upgraded.yaml

```
@@ -0,0 +1,17 @@
+version: 1.0
+name: Initialized Upgraded Proxy L1
+priority: P2
+addresses:
+  - 0x9ba6e03d8b90de867373db8cf1a58d2f7f006b3a
+  - 0xdE1FCfB0851916CA5101820A69b13a4E276bd81F
```

Could we add a comment above each line indicating which contract the address represents, like this:

```suggestion
  # AddressManager
  - 0xdE1FCfB0851916CA5101820A69b13a4E276bd81F
```

---

Nice idea! I will make this one as a standard!